uk.co.compucorp.civicrm.tasksassignments/css/civitasks.css

```diff
@@ -12853,15 +12853,17 @@ fieldset[disabled]
   border: 0;
   padding: 10px !important;
 }
-.mobile [type='date'][uib-datepicker-popup] {
+#civitasks .___outside-namespace .mobile [type='date'][uib-datepicker-popup], #cividocuments .___outside-namespace .mobile [type='date'][uib-datepicker-popup] {
   line-height: normal;
+  /* stylelint-disable selector-max-compound-selectors */
+  /* stylelint-enable */
```

Comment:
what caused this file to change?

Reply:
Really thanks for noticing this bug. I have created a ticket PCHR-3724, to fix this across all repos. Please check the ticket for more details.